recipes/tributary/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "tributary" %}
+{% set version = "0.1.0rc2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 11d15f1eaeeb6c27247dc7706175334aaee06cc9b023ea4a8cc107e220a4b877
+
+build:
+  number: 0
+  skip: True  # [py<37 or (win)]
```

Comment:
Is seems like this can be a noarch python package

Reply:
What about arch-dependent dependencies?

Reply:
Please see the history https://github.com/conda-forge/staged-recipes/pull/10924/commits/1927c3fbe3c57951d845b2d898b9fc35a408f859

Reply:
Ahh yeah, arch dep deps, need it to be archful